modules/authentication/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "authentication",
+  "version": "1.0.1",
+  "description": "User authentication module",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Treehouse-Gaming/treetalk.git"
+  },
+  "author": "Julian Biermann",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Treehouse-Gaming/treetalk/issues"
+  },
+  "dependencies": {
+    "crypto-util": "file:../crypto-util",
+    "user": "file:../user"
```

Comment:
Der Filelink zu den Modulen wird bereits in der Main package.json gesetzt, daher muss in den modulen nur noch über den namen und die benötigte version referenziert werden (ausserdem muss so der pfad zum modul nur einmal angegeben werden):
"crypto-util": "*",
 +    "user": "*"